sound/soc/sof/pm.c

```diff
@@ -250,6 +250,26 @@ static int sof_suspend(struct device *dev, bool runtime_suspend)
 	return ret;
 }
 
+int snd_sof_dsp_power_down_notify(struct snd_sof_dev *sdev)
+{
+	int ret;
+	/* Notify DSP of upcoming power down */
+	if (sof_ops(sdev)->remove) {
+		ret = sof_send_pm_ctx_ipc(sdev, SOF_IPC_PM_CTX_SAVE);
```

Comment:
You could simplify this function. Just retune aof_se d_pm_ctx_save() here

Reply:
Done